app/presenters/coronavirus/page_presenter.rb

```diff
@@ -11,6 +11,7 @@ def payload
     {
       "base_path" => path,
       "title" => title,
+      "description" => "Find information on COVID-19, including guidance and support.",
```

Comment:
I think this can be replaced with `details["description"]` as I think `corona_content` is an object of type `ContentBuilder` where you've already added the description content.